### ISSUE TITLE
Address warnings during Gradle build

### ISSUE THREAD

**Issue by HEdingfield** [BrightSpots/rcv#280]
It still works fine, but shouldn't be too hard to get this figured out:

> -> Task :compileJava
> C:\Users\MyName\IdeaProjects\rcv\src\main\java\com\rcv\GuiConfigController.java:528: warning: [unchecked] unchecked call to SimpleObjectProperty(T) as a member of the raw type SimpleObjectProperty
>           return new SimpleObjectProperty(checkBox);
> 
>   where T is a type-variable:
>     T extends Object declared in class SimpleObjectProperty
> C:\Users\MyName\IdeaProjects\rcv\src\main\java\com\rcv\GuiConfigController.java:518: warning: [unchecked] unchecked method invocation: method setCellValueFactory in class TableColumn is applied to given types
>     tableColumnCandidateExcluded.setCellValueFactory(
> 
>   required: Callback<CellDataFeatures<S,T>,ObservableValue<T>>
>   found: Callback<CellDataFeatures<Candidate,Boolean>,ObservableValue<Boolean>>
>   where S,T are type-variables:
>     S extends Object declared in class TableColumn
>     T extends Object declared in class TableColumn
> 2 warnings

**Comment by HEdingfield** [comment]
Note: this is still an issue after updating to the latest version of Gradle. Can repro by running `clean` first before `run` or `test`.

**Comment by HEdingfield** [comment]
Fixed with #462!

**Comment by HEdingfield** [comment]
Rolled back part of #462 in #493 because, although it resolved the warning mentioned in #304, it also broke checkboxes in the Candidates table (so they only appear as "true" or "false" and are uneditable).

Marking #304 as a duplicate of this, since it's specifically just a reminder to also clean up this line in `GuiConfigController.initialize()`:
```
// noinspection unchecked
return new SimpleObjectProperty(checkBox);
```

**Comment by tarheel** [comment]
Rendered moot by #519 and #522, which disabled editing in these tables.
